#2021-11-16
link0_T_robot: [1., 0., 0., 0.,
              0., 1., 0., 0.,
              0., 0., 1., 0.,
              0., 0., 0., 1.]

cam0:
   base_T_rgb_down: [-0.204601556861,-0.205813131165,0.956963509215,0.718200461496,
-0.978812971907,0.0509690350479,-0.198311178436,0.790861017514,
-0.00796046206186,-0.977263072313,-0.211880906496,0.267983455978,
0.0,0.0,0.0,1.0]
   base_T_rgb_up: [-0.190854524928,0.267037086265,0.944598192288,0.686835715044,
-0.981609494908,-0.0478358566668,-0.184809443272,0.799125889285,
-0.00416531151155,-0.962498272921,0.271255829069,0.285359767662,
0.0,0.0,0.0,1.0]
   depth_T_rgb: [0.999938,0.009284,-0.006203,-0.003212,
-0.009103,0.999549,0.028629,-0.065801,
0.006466,-0.028571,0.999571,-0.008343,
0.0,0.0,0.0,1.0]
   base_T_cam_down: [-0.212435585732,-0.176460589741,0.961109977681,0.713925375673,
-0.977048293757,0.0541784381065,-0.206011262338,0.789568981839,
-0.0157187745345,-0.982815175637,-0.183920296198,0.201728298871,
0.0,0.0,0.0,1.0]
   base_T_cam_up: [-0.194222676334,0.295696898859,0.935329178727,0.713472474787,
-0.980845717302,-0.0441698236611,-0.18971047717,0.791486239763,
-0.0147836781288,-0.954259756582,0.298611732918,0.225012353933,
0.0,0.0,0.0,1.0]
   cambase_p_Kdepth: [0.0, 0.0, 0.0018]
   cambase_q_Kdepth: [0.525483, -0.525483, 0.473147, -0.473147]
   Kdepth_p_rgb: [0.032079, 0.0022643, -0.0038309]
   Kdepth_q_rgb: [0.050553, -7.6103e-5, -0.0025044, 0.99872]
cam1:
   base_T_rgb_down: [0.0932824294553,-0.219888129673,0.971054889686,0.717926556826,
-0.995301997644,-0.0459942056666,0.0851966345062,-0.83841258229,
0.0259291696938,-0.974440220575,-0.223145545967,0.282380656745,
0.0,0.0,0.0,1.0]
   base_T_rgb_up: [0.0966502073887,0.302770615877,0.948150141894,0.684563319457,
-0.995197653784,0.0145583914129,0.0967971236216,-0.845416001138,
0.0155037838501,-0.952952258721,0.302723678105,0.304527796036,
0.0,0.0,0.0,1.0]
   depth_T_rgb: [0.999993,0.001995,0.00321,-0.002708,
-0.00202,0.999967,0.007836,-0.057032,
-0.003195,-0.007842,0.999964,-0.005981,
0.0,0.0,0.0,1.0]
   base_T_cam_down: [0.0959608978792,-0.212460655805,0.972446704984,0.711885566558,
-0.995112955092,-0.0433145361688,0.0887335542054,-0.843046947412,
0.0232686164826,-0.976209422361,-0.215578422016,0.22547911784,
0.0,0.0,0.0,1.0]
   base_T_cam_up: [0.100297757807,0.309994838067,0.945433102132,0.708169186774,
-0.994850570003,0.0173267797837,0.0998584162079,-0.84652462239,
0.0145746001397,-0.950580611358,0.310136808234,0.252208678876,
0.0,0.0,0.0,1.0]
   cambase_p_Kdepth: [0.0, 0.0, 0.0018]
   cambase_q_Kdepth: [0.525483, -0.525483, 0.473147, -0.473147]
   Kdepth_p_rgb: [0.032022, 0.0027479, -0.0038372]
   Kdepth_q_rgb: [0.048609, 0.0020356, 0.0030127, 0.99881]